package.json

```diff
@@ -29,9 +29,9 @@
     "danger": "^3.1.7",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
-    "eslint": "^4.19.1",
+    "eslint": "^5.1.0",
```

Comment:
I'm hesitant about this change, `eslint-config-airbnb@17.0.0` has a peer dependency of `4.19.1` which is why is didn't upgrade all the way to `5.1.0` in my last PR, see https://github.com/airbnb/javascript/blob/cae920fc308325b40f49c5e8f76bbeb5d8663fc5/packages/eslint-config-airbnb/package.json#L75